Examples/Scripts/Python/material_recording.py

```diff
@@ -87,4 +90,4 @@ def runMaterialRecording(g4geo, outputDir, tracksPerEvent=10000, s=None):
     )
     g4geo = acts.examples.geant4.dd4hep.DDG4DetectorConstruction(dd4hepSvc)
 
-    runMaterialRecording(g4geo=g4geo, tracksPerEvent=10, outputDir=os.getcwd()).run()
+    runMaterialRecording(g4geo=g4geo, tracksPerEvent=100, outputDir=os.getcwd()).run()
```

Comment:
I'm guessing with the changes from this PR, the runtime of the example doesn't increase drastically with this?

Reply:
The pytest for this example set those value so it shouldn't affect it. I changed this in preparation of moving the mapping doc to the python binding (those value are what you need for a good map). Since people often ask me how many event they need.